spec/jobs/async_batch_destroy_job_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+RSpec.describe AsyncBatchDestroyJob, type: :job do
+  describe '#perform' do
+    include_context :init_rateplan
+    include_context :init_destination, id: 1, initial_rate: 0.3
+    include_context :init_destination, id: 2, initial_rate: 0.5
+    include_context :init_destination, id: 3, initial_rate: 0.7
```

Comment:
It would be nice to have more then 1000 records in this test

Reply:
I didn't want to overload test with too many records so I tested multiple batches case by lowering BATCH_SIZE constant.
The problem is if this case crashes we won't know it until we do the same thing again. So I agree that this test should be reworked.

It can be a good idea to mock BATCH_SIZE constant. Seems like killing two birds with one stone.